package.json

```diff
@@ -31,21 +31,23 @@
     "downshift": "^3.4.8"
   },
   "peerDependencies": {
-    "react": "^16.0.0",
-    "react-dom": "^16.0.0",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
```

Comment:
Are we going to need to upgrade the React version in the app if we are making 16.12 a peer dep?

Reply:
yup, I figured we can do this, give it a go and if the upgrade is causing issues in maps we can revert. But there should be no issues upgrading to 16.12.

Reply:
yeah shouldnt be issues just need to make sure we do bump the maps version

Reply:
https://edozohq.atlassian.net/browse/MAPS-96?atlOrigin=eyJpIjoiNWMzMTdjY2UzY2NhNGZmMWIyYmIxZmMyZTBkMDJkODIiLCJwIjoiaiJ9